src/fides/ctl/core/config/__init__.py

```diff
@@ -184,21 +111,26 @@ def censor_config(config: FidesConfig) -> Dict[str, Any]:
 @lru_cache(maxsize=1)
 def get_config(config_path_override: str = "", verbose: bool = False) -> FidesConfig:
     """
-    Attempt to load user-defined configuration.
+    Attempt to load user-defined configuration file, otherwise will use defaults
+    while still respecting any env vars set by the user.
 
-    This will fail if the first encountered configuration file is invalid.
+    Order of file checks is:
+        1. The config path provided by the user to the CLI
+        2. The config path provided by the user via an env var
+        3. The default config path
 
-    On failure, returns default configuration.
+    This will fail if the first encountered configuration file is invalid.
```

Comment:
Great update for this docstring. This will help future developers make sense of the order of operations for loading the config.